ts/index.d.ts

```diff
@@ -199,6 +200,7 @@ export declare function login({
   verificationCode?: string;
   channel?: "sms" | "email";
   redirect?: string | boolean;
+  options?: object;
```

Comment:
Added new param to TS definitions

Reply:
Thanks! I wonder if there's an easy way to auto-generate these from JSDocs.